tests/observability/metrics/utils.py

```diff
@@ -1202,13 +1202,15 @@ def compare_metric_file_system_values_with_vm_file_system_values(
 
 
 def expected_metric_labels_and_values(
-    prometheus: Prometheus, metric_name: str, expected_labels_and_values: dict[str, str]
+    expected_labels_and_values: dict[str, str], values_from_prometheus: dict[str, str]
 ) -> None:
```

Comment:
This function is also used in `test_metric_kubevirt_cdi_storageprofile_info`, the change will break that test